README.md

```diff
@@ -28,12 +28,53 @@ open localhost:1313
 
 ## [Contributing](https://github.com/breadtubetv/breadtubetv/blob/master/CONTRIBUTING.md)
 
-### Adding a Channel
+### Configuring the `bake` CLI
+
+The `.bake.yaml` configuration file can be stored in the following locations:
+
+- `$HOME/.bake.yaml`
+- `./bake.yaml` (In other words, the current directory from which you're running the CLI)
 
-#### [Walkthrough Video](https://youtu.be/jpOun7YXFpg)
+Current configuration options and default values:
+
+- `channelsDir: "../data/channels"` : Set to the location of the `channels` directory under the `data` directory. E.g. `$GOPATH/src/github.com/breadtubetv/breadtubetv`
```

Comment:
```suggestion
- `channelsDir: "../data/channels"` Directory of channel data files.    
  E.g. `$GOPATH/src/github.com/breadtubetv/breadtubetv/data/channels`
```

Was going to suggest defining the channelPagesDir, but maybe that setting doesn't need to be a part of this PR?

I'm really happy for this, it will help me out a lot, but maybe a further refactor will just be setting the breadtube root directory, rather than having configs for each types.

Thinking for example when we add video tasks, we're going to want the videos to be in the same relative location as the channels.